HW_9/task1.py

```diff
@@ -0,0 +1,63 @@
+"""Напишите следующие функции:
+○ Нахождение корней квадратного уравнения
+○ Генерация csv файла с тремя случайными числами в каждой строке.
+100-1000 строк.
+○ Декоратор, запускающий функцию нахождения корней квадратного
+уравнения с каждой тройкой чисел из csv файла.
+○ Декоратор, сохраняющий переданные параметры и результаты работы
+функции в json файл."""
+import csv
+import json
+import random
+from typing import Callable
+
+CSV_FILE_NAME = 'test_csv.csv'
+JSON_FILE_NAME = 'test_json.json'
+
+
+def decor_run_for_csv_data(func: Callable):
+    def wrapper():
+        res = []
+        with (open(CSV_FILE_NAME, 'r', newline='\n', encoding='utf-8') as f_csv):
```

Comment:
Если вы открываете один файл, то скобки не нужны.
```suggestion
        with open(CSV_FILE_NAME, 'r', newline='\n', encoding='utf-8') as f_csv:
```